testplan/web_ui/testing/src/AssertionPane/AssertionTypes/AttachmentAssertions.js

```diff
@@ -45,7 +23,7 @@ const theme = createTheme({
  */
 export const AttachmentAssertion = (props) => {
   const content = getAttachmentContent(props.assertion, props.reportUid);
-  return <ThemeProvider theme={theme}>{content}</ThemeProvider>;
+  return content;
```

Comment:
eh... but then the attachment content losts its style?

Reply:
No, the main styling from the assertions is applied. I can share a session tomorrow to demonstrate.